packages/wdio-cucumber-framework/src/index.ts

```diff
@@ -38,13 +42,20 @@ import { DEFAULT_OPTS } from './constants.js'
 import { setUserHookNames } from './utils.js'
 import type { CucumberOptions, StepDefinitionOptions, HookFunctionExtension as HookFunctionExtensionImport } from './types.js'
 
+import TagExpressionParser from '@cucumber/tag-expressions'
+
+const uuidFn = IdGenerator.uuid()
+const builder = new Gherkin.AstBuilder(uuidFn)
+const matcher = new Gherkin.GherkinClassicTokenMatcher()
+const gherkinParser = new Gherkin.Parser(builder, matcher)
+
 const require = createRequire(import.meta.url)
 const EventDataCollector = require('@cucumber/cucumber/lib/formatter/helpers/event_data_collector').default
```

Comment:
It seems Cucumber now has ESM support so maybe we can update this and now import the `EventDataCollector` via import statement

Reply:
Don't think this is possible yet. See https://github.com/TypeStrong/ts-node/pull/1585
Since `EventDataCollector` is still not exporterdfrom `@cucumber/cucumber` I need to
```ts
import { EventDataCollector } from '@cucumber/cucumber/lib/formatter/helpers'
// or
import { EventDataCollector } from '@cucumber/cucumber/lib/formatter/helpers/index.js'

```
But I'm getting 
```shell
No known conditions for "./lib/formatter/helpers" specifier in "@cucumber/cucumber" package
```

Reply:
Cheers! I saw an export for all lib files but then realised that it was only for `require` imports:

```json
    "./lib/*": {
      "require": "./lib/*.js"
    },
```

😞